core/src/main/scala/org/apache/spark/scheduler/SparkListenerBus.scala

```diff
@@ -71,7 +71,6 @@ private[spark] trait SparkListenerBus
         listener.onNodeUnblacklisted(nodeUnblacklisted)
       case blockUpdated: SparkListenerBlockUpdated =>
         listener.onBlockUpdated(blockUpdated)
-      case logStart: SparkListenerLogStart => // ignore event log metadata
```

Comment:
I do not see it getting consumed apart from registering some metadata, so I guess it should be fine as this event already logs the version